lib/node/node.ml

```diff
@@ -3,6 +3,7 @@ open Types
 
 module Message = Message
 module Client = Client
+module Failure_detector = Failure_detector
```

Comment:
This is not working as expected. I thought I would be able to do `open Pollinate.Node.Failure_detector` in the tests suite, but nope...